### ISSUE TITLE
Windows Terminal crashes (Vanishes with no errors) when large font sizes are used

### ISSUE THREAD

**Issue by feoh** [microsoft/terminal#18182]
### Windows Terminal version

1.21.2911.0

### Windows build number

10.0.22631.0

### Other Software

The bug sometimes manifests when I'm running Neovim although it's also quite common from just the terminal itself, either in WSL2 running stock Ubuntu or in Powershell on the native Windows host side.

### Steps to reproduce

Open Terminal. Hit Ctrl+ 5 times to boost font size to my ludicrously large default :)

Open a text based editor, stream some text to the terminal. I can't really pin down the behavior that triggers it very well or I'd take a screen recording.

This issue has happened in the past but was fixed and vanished for a while. I had previously been tracking it in https://github.com/microsoft/terminal/issues/8730

### Expected Behavior

I expect Terminal not to just vanish taking my editing session, command session or whatever else I'm working on with it. This is the worst kind of failure in a terminal program IMO.

Even if it just hard crashed I could maybe at least just copy what my current state was, vanishing without a trace is tough.

### Actual Behavior

Terminal vanishes with no error in mid stream, sometimes while typing, sometimes while displaying streamed text.

**Comment by carlos-zamora** [comment]
Thanks for filing! Could you please capture a dump file using this guide /? https://github.com/microsoft/terminal/wiki/Troubleshooting-Tips#capture-automatically

**Comment by feoh** [comment]
Happy to. It's frustratingly irregular, it won't happen for a while, and then the window will vanish very shortly after opening a new tab.

Thanks for taking a look!

**Comment by DHowett** [comment]
If you have any historical entries in your Event Viewer for this crash, that may help too! :)

**Comment by feoh** [comment]
@DHowett Any clues for the clueless around what I'd be searching for? Pardon I'm a bit Event Viewer clueless.

Thanks so much again for taking a look.

**Comment by lhecker** [comment]
> Any clues for the clueless around what I'd be searching for? Pardon I'm a bit Event Viewer clueless.

Sorry for not responding earlier. When an application crashes it should look like this:
![Image](https://github.com/user-attachments/assets/79641bee-9aa8-4f7a-927a-2f8bddcb8a28)

It seems we haven't mentioned it yet, but please ensure you're using the very latest GPU drivers, straight from the Intel/AMD/Nvidia website. (If you don't know where to find them, let me know what GPU you're using and I'll find it for you.)

**Comment by feoh** [comment]
After some significant local testing I've determined that the problem was actually a corrupted Ubuntu 20.04 image that was causing the crash.

Sorry about the mis-identified bug folks and thank you for your time and attention! Windows Terminal and the new console host are *amazing*!

**Comment by DHowett** [comment]
Wow! I'm not sure how that all fit together, but thank you so much for root causing it and coming back to tell us! 🙂

**Comment by feoh** [comment]
Absolutely.

My reasoning:

* I noticed that the vanishing window ONLY happened when I was trying to use my Ubuntu 20.04 WSL2 distro
* I had the idea to try and see if the failure happened when I invoked the distro from the Powershell `wsl` command, which it did. So I could run `wsl -d Ubuntu-20.04` or similar, and after a couple of command line invocations the WSL session would terminate and I'd be back in Powershell.
* Given the above, I was reasonably sure the distro was faulty. So I blew it away completely and re-installed from scratch (This is a special purpose distro I'm only using to work on OpenEdX legacy issues so no harm done)
* The issue stopped recurring after nuke & pave
